src/server.ts

```diff
@@ -0,0 +1,49 @@
+import dotenv from "dotenv";
+import { Config } from "./config.js";
+import ExpressConfig from "./express/express.config.js";
+import { ApiManager } from "./chain_service/api.js";
+import Faucet from "./chain_service/faucet.js";
+import { AddressError, RateError } from "./chain_service/faucetErrors.js";
+import { SlackNotifier } from "./slack_service/slack.js";
+
+dotenv.config();
+const PORT = process.env.PORT || 3000;
+
+// Load base config and initializations
+const config = new Config("config.json");
+const apiManager = new ApiManager(config);
+const slackNotifier = new SlackNotifier();
+const faucet = new Faucet(config, slackNotifier, apiManager);
+await faucet.init();
+const app = ExpressConfig(config.getRateLimitConfig());
+
+// fund route
+app.get("/fund", async (req, res) => {
```

Comment:
We should probably describe how one can use the faucet to receive funds in the README. It would also be nice if we can ignore the capitalization of the `to` query parameter. At the moment, the capitalization matters and `/funds?TO=...` would not work.

Reply:
Sure thing.